forwards_paper.tex

```diff
@@ -196,14 +196,40 @@ \section*{Results}
 %%%%%%
 \subsection*{Simulation benchmarks}
 
-\plr{Jaime and Kevin to describe simulation methods here (separately).}
-
-Comparison of simulation with/without msprime, using \simupop{}
-or maybe just a simple haploid simulation with 1000 QTL and stabilizing selection on a trait (say).
-
-Maybe an estimate of how long \emph{just} the ARG recording and simplification takes,
-so that then we can say how fast the simulator would have to be to do $10^6$ whole chromosomes for $10^7$ generations
-in a day.
+To measure the performance gains from ancestry recording, we ran simulations implemented in C++ (see above) with and
```

Comment:
Where is the "(see above)"?

Reply:
I guess it should be "see below", referring to lines 927-941.  I'll fix that.